pkg/graph/schema/types/model_plan_translation.graphql

```diff
@@ -0,0 +1,13 @@
+
+"""
+Represents model plan base translation data
+"""
+type ModelPlanTranslation {
+  modelName: TranslationField! @goTag(key: "db", value: "model_name")
+  previousName: TranslationField! @goTag(key: "db", value: "previous_name")
```

Comment:
previousName and nameHistory aren't actually fields in the database, they are GQL resolvers, I don't think it hurts to leave them though. I wonder if we should remove the @goTag directives for these fields to be clear about that?

Reply:
Hmm yeah, does that affect being able to translate the data at all? If not, we can remove them

Reply:
It wouldn't affect it in this instance, because we are programmatically accessing by DB fields. (And since the fields don't exist, we'd never hit those data points.